talk-4-advanced-js-2/README.md

```diff
@@ -11,8 +11,13 @@
 ### Callbacks
 - 
 
+### TCO
+- As it stands, JavaScript is not tail-call optimized for recursive calls. ES6 stands to change that, but it's still something to keep in mind with recusive functions in JavaScript, which, as it stands, is not tail-call optimized for recursive calls. ES6 stands to change that, but it's still something to keep in mind with recusive functions in JavaScript, which, as it stands, is not tail-call optimized for recursive calls. ES6 stands to change that, but it's still something to keep in mind with recusive functions in `Maximum call stack size exceeded`
```

Comment:
Actually now that I'm reading it again, is it all a play on the fact that you're talking about recursion? That went right over my head :sob: